.github/mvn-settings.xml

```diff
@@ -124,5 +124,6 @@
   </profiles>
   <activeProfiles>
     <activeProfile>google-mirror</activeProfile>
+    <activeProfile>snapshots</activeProfile>
```

Comment:
I guess you plan to remove these commits when it's ready?
Wondering if we shouldn't have a dedicated profile, makes it easier next time one needs something similar.